app/views/notifications/_notification.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= link_to notification_path(notification),
+            data: { turbo_method: :patch },
+            class: [
+              # base card
+              "group flex items-center gap-4 p-4 rounded-lg transition",
+              "border",               
+              "hover:bg-gray-100 dark:hover:bg-gray-700",
+              "focus:outline-none focus:ring-2 focus:ring-blue-400",
+
+              # read vs unread
+              notification.read? ?
+                "bg-white dark:bg-gray-800 border-gray-200 dark:border-gray-700" :
+                "bg-gray-100 dark:bg-gray-900 border-gray-500"
+            ].join(" ") do
+%>
+  <svg xmlns="http://www.w3.org/2000/svg"
```

Comment:
instead of putting svgs directly in the html files, I prefer using the `inline_svg_tag` pattern